src/groups/mqb/mqba/mqba_domainmanager.cpp

```diff
@@ -657,6 +719,8 @@ void DomainManager::createDomain(
     const bsl::string&                         name,
     const mqbi::DomainFactory::CreateDomainCb& callback)
 {
+    // executed by *ANY* thread
+
     if (!d_isStarted) {
```

Comment:
`createDomain` doesn't alter the `DomainManager`'s state in the function's body, it only looks up the `d_isStarted` field.
It also possibly calls `d_domainResolver_mp->locateDomain` with `DomainManager::onDomainResolverDomainLocatedCb` callback, both of these functions are protected by mutexes.
So I've added a description `executed by *ANY* thread` since it's safe.

Reply:
To improve readability, can you add the `executed by *ANY* thread` description to related functions including `DomainResolver::locateDomain()`, `DomainResolver::getOrReadDomain()` and `DomainResolver::getOrRead()`?